gatsby-config.js

```diff
@@ -64,6 +65,37 @@ module.exports = {
             'python',
           ],
         },
+        relatedResources: {
+          swiftype: {
+            resultsPath: `${__dirname}/src/data/swiftype-resources.json`,
+            engineKey: 'Ad9HfGjDw4GRkcmJjUut',
+            refetch: Boolean(process.env.BUILD_RELATED_CONTENT),
+            filter: ({ slug }) =>
+              slug ===
+              '/docs/integrations/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer',
```

Comment:
Oops, will update this. I did this for testing purposes so that I wouldn't trigger a full rebuild

Reply:
Actually, I'm going to leave this in for now so that we can test the build end-to-end once or twice with related resources to make sure its all hooked up correctly. Once we confirm, then I'll update this to be the real logic.